src/extensions/scratch3_microbit/index.js

```diff
@@ -438,7 +656,11 @@ class Scratch3MicroBitBlocks {
                 '---',
                 {
                     opcode: 'whenPinConnected',
-                    text: 'when pin [PIN] connected',
+                    text: formatMessage({
+                        id: 'microbit.whenPinConnected',
+                        default: 'when pin [PIN] connected',
+                        description: 'when a pin is connected to the micro:bit'
```

Comment:
This is a difficult one to describe. I think we'll get better translations with a description like:
when the pin detects a connection to Earth/Ground